prow/crier/controller.go

```diff
@@ -84,9 +84,6 @@ func (r *reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 	log := logrus.WithField("reporter", r.reporter.GetName()).WithField("key", req.String()).WithField("prowjob", req.Name)
 	log.Debug("processing next key")
 	result, err := r.reconcile(ctx, log, req)
-	if err != nil {
```

Comment:
How is this duplicated? It might be duplicated with the controller-runtime reporting but that doesn't have all the fields or WDYM?

Reply:
actually nvm, I didn't realize that `reconcile` returns error at multiple places, will restore this

Reply:
done